components/explore/ExploreSort.vue

```diff
@@ -127,21 +101,16 @@ function onChange(selected) {
 watch(
   () => route.query.sort,
   (sort) => {
-    selectedSort.value = sort as string[]
+    if (sort?.length) {
+      selectedSort.value = Array.isArray(sort)
+        ? (sort as string[])
+        : ([sort] as string[])
+    } else if (selectedSort.value.length) {
+      selectedSort.value = []
+    }
   },
+  { immediate: true },
```

Comment:
checking

Reply:
regarding  `{ immediate: true }` seems fine , checking further if it's causing any issue 

I think this might be the problem, it's happening on canary 

https://github.com/kodadot/nft-gallery/assets/44554284/d74f7ce1-bd51-43b5-98bf-d95e83d41be8

Reply:
Please do open a new issue :)